tool/gfm_tests.json

```diff
@@ -5461,7 +5461,7 @@
     "example": 605,
     "start_line": 9025,
     "end_line": 9029,
-    "section": "Autolinks (extension)",
+    "section": "Autolinks Email (extension)",
```

Comment:
FYI: this is generated from the GFM source. Shouldn't hand edit.

Reply:
Alright, I can definitely revert that. It was just for testing to cut down on the number of tests being run.

Reply:
Please! I just checked the latest GFM output – these haven't changed

Reply:
I don't think that this is quite ready to merge yet, so I'd like to leave the modified tests as is for now to ease my dev cycle.

I'm still trying to figure out what is causing it to fail some of the Autolink (extension) tests. I'm not seeing why it's failing [example-598](https://github.com/dudeofawesome/dart-lang-markdown/blob/0a503e882d593f6402b215de9e0712e7b40183f8/tool/gfm_tests.json#L5382) and [example-599](https://github.com/dudeofawesome/dart-lang-markdown/blob/0a503e882d593f6402b215de9e0712e7b40183f8/tool/gfm_tests.json#L5393). I also am not sure how to fix [example-604](https://github.com/dudeofawesome/dart-lang-markdown/blob/0a503e882d593f6402b215de9e0712e7b40183f8/tool/gfm_tests.json#L5448).

Reply:
Please revert the changes to this file. It's autogenerated...